node/rpc.go

```diff
@@ -0,0 +1,133 @@
+package node
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/log"
```

Comment:
use our internal log?

Reply:
I see our code also use eth log. which log do you prefer?

Reply:
try https://github.com/harmony-one/harmony/commit/83bd41fb930dd3822f2ceb9d3b22bbde7dbc375f